recipes/tcb-span/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1)
+
+set(CMAKE_CXX_STANDARD 17)
+
+project(test_package)
+
+find_package(tcb-span REQUIRED)
+
+add_executable(test_package test_package.cxx)
+target_link_libraries(test_package tcb-span::tcb-span)
```

You are missing our cmake settings injection + C++17 is not understood until CMake 3.8

```suggestion
cmake_minimum_required(VERSION 3.8)
project(test_package)

include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
conan_basic_setup()

find_package(tcb-span REQUIRED)

add_executable(test_package test_package.cxx)
target_link_libraries(test_package tcb-span::tcb-span)
set_property(TARGET test_package PROPERTY CXX_STANDARD 17)
```

---

Ahh yes, I thought about it and then forgot :)

BTW, can I
```
include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
conan_basic_setup()
```

In my projects too, or do I need anything special?
I had some troubles, and in the end I used the "manual" approach of setting `CMAKE_BIN_PATH` (and the other too).

---

It depends on the "call entry point" my OSS I typically build with the conan cli and I do include this file (so my conan profile and settings and injected into the build) however if you take the pure CMake approach you can just use the `"cmake_find_package[_multi]"` generators and not include this file

---

Yes, I typically use de `cmake_find_package` generator, so that CMake is completely isolated of Conan (except for 2 lines setting the paths).